git/test/fixtures/blame_incremental_2.11.1_plus

```diff
@@ -0,0 +1,33 @@
+82b8902e033430000481eb355733cd7065342037 2 2 1
```

Comment:
I don't think there should be empty lines in the fixture.

If it helps I generated the fixture with `git blame --incremental -p 82b8902e033430000481eb355733cd7065342037 -- AUTHORS > git/test/fixtures/blame_incremental_2.11.1_plus` (which I believe matches how `blame_incremental` was generated).